docs/product/security-policy-reporting.mdx

```diff
@@ -19,7 +19,9 @@ To configure CSP reports in Sentry, you’ll need to send a header from your ser
 ```
 Content-Security-Policy: ...;
     report-uri https://___ORG_INGEST_DOMAIN___/api/___PROJECT_ID___/security/?sentry_key=___PUBLIC_KEY___;
-    report-to {"group":"default","max_age":10886400,"endpoints":[{"url":"https://___ORG_INGEST_DOMAIN___/api/___PROJECT_ID___/security/?sentry_key=___PUBLIC_KEY___"}],"include_subdomains":true}
+    report-to csp-endpoint
+
+Report-To: {"group":"csp-endpoint","max_age":10886400,"endpoints":[{"url":"https://___ORG_INGEST_DOMAIN___/api/___PROJECT_ID___/security/?sentry_key=___PUBLIC_KEY___"}],"include_subdomains":true}
```

Comment:
I thought the issue was that query parameters (e.g. `?sentry_key=___PUBLIC_KEY___`)  aren't supported? But if this now works, that's great.

Reply:
This is tested and it works